reservation/views.py

```diff
@@ -34,14 +33,11 @@ def select_seats(request, movie, schedule):
     movie_object = get_object_or_404(Movie, pk=movie)
     schedule_object = get_object_or_404(Schedule, pk=schedule)
 
-    hall = Schedule.objects.values_list('hall', flat=True).get(id=schedule)
+    hall = Schedule.objects.values_list('hall', flat=True).get(id=schedule_object.id)
```

Comment:
hall can be directly accessed as well from the schedule_object:
hall = schedule_object.hall or something similar.
I've added another message but it got away somehow. Maybe I didn't add it, after all :)